tools/src/wyvern/tools/types/UnresolvedType.java

```diff
@@ -99,6 +100,11 @@ public String getName() {
 	@Override
 	public ValueType getILType(GenContext ctx) {
 		// TODO Auto-generated method stub
-		return null;
+		if(typeName.equals("Int")) {
```

Comment:
Do we need this?  Or can we set up a Context that maps Int to system.Int in the prelude, so that ctx.getType() and the code below just does this automatically?